lectures/inflation_history.md

```diff
@@ -557,36 +606,37 @@ lab = ['Price Index (Marks or converted to Marks)',
        '1/Cents per Mark (or Reichsmark converted to Mark)']
 
 # create plot
-fig, ax = plt.subplots(figsize=[10,7], dpi=200)
+fig, ax = plt.subplots(dpi=200)
 ax1 = pe_plot(p_seq, e_seq, df_Germ.index, lab, ax)
 
-plt.figtext(0.5, -0.02, 'Germany', 
-            horizontalalignment='center', 
-            fontsize=12)
 plt.show()
 ```
 
 ```{code-cell} ipython3
+---
+mystnb:
+  figure:
+    caption: "Monthly inflation rate (Germany)"
+    name: inflationrate_germany
+---
 # plot moving average
-fig, ax = plt.subplots(figsize=[10,7], dpi=200)
+fig, ax = plt.subplots(dpi=200)
 _ = pr_plot(p_seq, df_Germ.index, ax)
 
-plt.figtext(0.5, -0.02, 'Germany', 
-            horizontalalignment='center', 
-            fontsize=12)
 plt.show()
 ```
 
 ## Starting and Stopping Big Inflations
 
-It is  striking  how **quickly**  (log) price levels in Austria, Hungary, Poland,
-and Germany leveled off after  rising so quickly.
+It is striking how **quickly** (log) price levels in Austria, Hungary, Poland, and Germany leveled off after rising so quickly.
```

Comment:
Bold is for defintions and italics is for emphasis: https://manual.quantecon.org/styleguide/writing.html#emphasis-and-definitions

Reply:
will change to `*`